modules/packages/Python.chpl

```diff
@@ -2622,6 +2622,23 @@ module Python {
     }
   }
 
+  private proc determineSliceBounds(bounds: range(?)): (Py_ssize_t, Py_ssize_t)
+    {
+    if (bounds.hasLowBound() && bounds.hasHighBound()) {
+      return (bounds.low.safeCast(Py_ssize_t),
+              bounds.high.safeCast(Py_ssize_t) + 1);
+
+    } else if (!bounds.hasLowBound() && bounds.hasHighBound()) {
+      return (min(Py_ssize_t), bounds.high.safeCast(Py_ssize_t) + 1);
+
+    } else if (bounds.hasLowBound() && !bounds.hasHighBound()) {
+      return (bounds.low.safeCast(Py_ssize_t), max(Py_ssize_t));
+
+    } else {
+      return (min(Py_ssize_t), max(Py_ssize_t));
+    }
+  }
+
```

Comment:
as a nit, I have a strong preference for this being explicitly marked `inline`

Reply:
further nit, the `{` should be attached to the end of the proc declaration, not on a new line

Reply:
I put it on a new line because it was over the 80 character limit with the declared return type

Reply:
But having it be inline will also impact that line length, so I'll reshuffle it (I don't like having `{` on separate lines on principle either)